arcade/text.py

```diff
@@ -583,18 +580,18 @@ def position(self, point: Union[Point, Point3]):
 
 
 def create_text_sprite(
-    text: str,
-    color: RGBA255 = arcade.color.WHITE,
-    font_size: float = 12,
-    width: int = 0,
-    align: str = "left",
-    font_name: FontNameOrNames = ("calibri", "arial"),
-    bold: bool = False,
-    italic: bool = False,
-    anchor_x: str = "left",
-    multiline: bool = False,
-    texture_atlas: Optional[arcade.TextureAtlas] = None,
-    background_color: Optional[RGBA255] = None,
+        text: str,
+        color: RGBA255 = arcade.color.WHITE,
+        font_size: float = 12,
+        width: int = 0,
+        align: str = "left",
+        font_name: FontNameOrNames = ("calibri", "arial"),
+        bold: bool = False,
+        italic: bool = False,
+        anchor_x: str = "left",
+        multiline: bool = False,
+        texture_atlas: Optional[arcade.TextureAtlas] = None,
+        background_color: Optional[RGBA255] = None,
```

Comment:
Knitpicking here, but these should probably not be indented. We've kind of used someting close to the black formatter everywhere else.